### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by pradeep10kumar** [UtrechtUniversity/yoda#338] [Q&A] [answered]
Dear Yoda team,

I need few clarifications:

1. Can Restricted (archived) data be made accessible by the Data Manager?
2.  How can Encrypted transportion of data be done?
3. Does Tool interface comply to WCAG version 2.0?
4. Does user retains ownership of research data in the system?

Thanks a lot in advance!

Best,
Pradeep

**Comment by lwesterhof** [comment, marked as answer]
Dear Pradeep,

1. Yes, datamanagers can change the metadata of data packages and with that the data access restriction.
2. You have several options depending on your start and end point. For example, you could use the iCommands for this.
3. No
4. This depends on the policy of your organization.

**Comment by pradeep10kumar** [comment]
Dear lwesterhof.

I appreciate your prompt response. I have a couple of follow-up questions for clarification:

1. I observed that when metadata is in the vault, even the data manager cannot alter or modify it. Is it feasible to modify metadata within the vault?
2. After a data package is published, is there a mechanism to unpublish it? If so, who has the authority to unpublish, and what is the process for doing so?

Best,
Pradeep

**Reply by lwesterhof** [reply]
Dear Pradeep,

1. Datamanagers can modify metadata in the vault. By navigating to the data package, clicking the metadata button followed by the update button they can make the changes.
2. Yes, the same mechanism used to publish can be used to depublish. See this sequence diagram: https://utrechtuniversity.github.io/yoda/design/overview/vault-space.html

**Comment by pradeep10kumar** [comment]


**Reply by lwesterhof** [reply]
1. Your service provider can help you with that.
2. That is by design, you do not have to be in a research group to be able to authenticate.

Note that this is for discussion and support on the product and not for support for a specific service.
